Align design delegates page - Closes #2167 #2347

massao opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### What issue have I solved?
<!--- Complementary description if needed -->
#2167
#2347 

### How have I implemented/fixed it?
<!--- Describe your technical implementation -->
- [x] Adjust table columns spacing;
- [x] Adjust search input size;
- [x] Adjust LSK signs;
- [x] Adjust cropped tooltip;
- [x] Disable register as delegate when using hwWallet.

Didn't change the things that were commented here: https://github.com/LiskHQ/lisk-hub/issues/2167#issuecomment-504386597

### How has this been tested?
<!--- Please describe how you tested your changes. -->
Delegates page should be as in the designs.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

osvaldovega commented on src/components/delegatesListView/delegateRow.js:
```diff
@@ -104,13 +104,13 @@ class DelegateRow extends React.Component {
         </div>
         <div className={`${columnClassNames.voteWeight} ${styles.weight} vote-weight`}>
           <LiskAmount val={data.vote} />
+          {' '}
```
do we need this empty spaces?

massao commented:
> 🐛 Added/Removed votes should have smaller height

Fixed in 51d322c3833839f7b722c01f12976688918026d3

> 🐛 3 Tooltips are back in design files as I can see. Can we request the text for the Forged LSK and implement it?

@reyraa is checking into this and should get back at us.

> 🐛 By LSK sign problem I meant that some columns have LSK sign in the header but some have LSK sign in the rows inside the table. Yashar had a decision where it should be placed

Talked with Yashar and implemented as suggested in c86e9dbb3cb5553a562a7ab4daf7af0cf85c2330

massao commented:
@Efefefef Updated the tooltips as described in https://github.com/LiskHQ/lisk-hub/issues/2347
